sqlalchemy_redshift/__init__.py

```diff
@@ -11,6 +11,26 @@
 __version__ = get_distribution('sqlalchemy-redshift').version
 
 from sqlalchemy.dialects import registry
+from sqlalchemy.dialects.postgresql.base import PGDialect
+import re
+
+def _get_server_version_info(self, connection):
+    v = connection.exec_driver_sql("select pg_catalog.version()").scalar()
+    m = re.match(
+            r".*(?:PostgreSQL|EnterpriseDB) "
+            r"(\d+)\.?(\d+)?(?:\.(\d+))?(?:\.\d+)?(?:devel|beta)?",
+            v,
+    )
+    if not m:
+        if "Redshift" in v: # There is no way of extracting Postgres version from the new version format
+            return (8, 0, 2)
```

Comment:
If we get word that the version string behavior change in Redshift is permanent, then I think we should just return this static (8, 0, 2) value always, rather than even attempting to parse the version string.